osgar/drivers/rosmsg.py

```diff
@@ -393,23 +389,23 @@ def parse_topic(topic_type, data):
         assert size == 40, size
         # __slots__ = ['score','calls','total_of_types']
         # _slot_types = ['int32','int32','int32[8]']        
-        return struct.unpack_from('<II', data, pos)  # only score and calls
+        return list(struct.unpack_from('<II', data, pos))  # only score and calls
```

Comment:
Frantisek asked why I change `()` for `[]` ... i.e. tuple and list. The reason is that `msgpack` is using only one type (interfaces several programming languages) and we decided to use lists. My change was motivated that I could replay module `rosmsg` and see "regression test".